examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -42,6 +42,22 @@ model:
   prompt_format: ???   # Options supported: ["canary", "canary2"]
   prompt_defaults: null
 
+  # Sub-config for specifying multiple metrics for multitask modeling.
+  # See `asr.collections.metrics.multitask` for detsils.
+  multitask_metrics_cfg:
+    log_predictions: true
+    metrics:
+      -   name: wer
+          _target_: nemo.collections.asr.metrics.WER
+          slots: 
+              task: "transcribe"
+      -   name: bleu
+          tokenize: flores101
```

Comment:
for current published models, doesn;t this need to be `bleu_tokenizer` ? You can add other options as a comment?